ex5/server/db/models/items.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const { Model } = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+	class Items extends Model {
+		/**
+		 * Helper method for defining associations.
+		 * This method is not a part of Sequelize lifecycle.
+		 * The `models/index` file will call this method automatically.
+		 */
+		static associate(models) {
+			// define association here
+		}
+	}
+	Items.init(
+		{
+			item_id: DataTypes.STRING,
```

Comment:
naming convention - fields should be camelCased, item_id should be itemId
also - item_id doesn't appear in the DB so this causes errors